overlays/default.nix

```diff
@@ -280,6 +280,8 @@ in
   telegram-desktop_git = callOverride ../pkgs/telegram-desktop-git { };
   tg-owt_git = callOverride ../pkgs/tg-owt-git { };
 
+  virtualboxKvm = nixpkgs.legacyPackages.${final.system}.virtualboxKvm;
```

Comment:
This makes no sense at all to add to the overlay. If you just want to cache, you probably want to add it to `_dev`. Using `nixpkgs.legacyPackages` is also the wrong way.

Reply:
Extending `packages` in `maintenance/dev/default.nix`.

Reply:
Oh, nice. I'll try to push that tomorrow - thanks for getting back at me 😁